cocos/core/animation/cross-fade.ts

```diff
@@ -81,8 +81,6 @@ export class CrossFade {
      * @param duration 切换时间。
      */
     public crossFade (state: AnimationState | null, duration: number) {
-        this._finished = false;
```

Comment:
BEFORE: Since `this.clear()` set `this._finished = true`, the `state` get no chance to update its weight.(`this.clear()` would be called if `duration === 0` or `this._managedStates.length === 0`).

To reproduce the issue:
- Switch the animation 3 times(Idle -> Walk -> Run -> Idle), in default duration.

- Change duration to `0` and switch, so that the bug.